uint32 id
bool led_on
bool button_pressed
float64 potentiometer_value
